tests/integration/docusaurus/connecting_to_your_data/connect_to_your_data_overview.py

```diff
@@ -1,21 +1,9 @@
-import pathlib
```

Comment:
This is a cleanup of a previous test script I wrote and was using as a template for this change.